graphql/tests/taskTests/results.json

```diff
@@ -38,6 +38,22 @@
           }
         }
       }
+    },
+    {
+      "query_file": "execution-task-group-id.graphql",
+      "result": {
+        "data": {
+          "taskTests": {
+            "testResults": [
+              {
+                "execution": 0,
+                "taskId": null,
+                "groupID": null
```

Comment:
Are these fields meant to return null?  i think you might be missing the resolver portion of this pr

Reply:
the fields are rolled up in the TestResult struct so there was no resolver code needed